ivy/functional/frontends/torch/spectral_ops.py

```diff
@@ -0,0 +1,19 @@
+import ivy
+from ivy.functional.frontends.torch.func_wrapper import (
+    to_ivy_arrays_and_back,
+)
+
+
+@to_ivy_arrays_and_back
+def hann_window(
+    window_length,
+    periodic=True,
+    *,
+    dtype=None,
+    device,
+    layout=ivy.strides,
+    required_grad=False
+) -> ivy.Array:
+    return ivy.hann_window(
+        window_length, periodic, dtype, layout, device, required_grad
```

Comment:
I don't know if I missed something but `ivy.hann_window()` does not seem to support `device` and `required_grad`